tensorflow_addons/optimizers/__init__.py

```diff
@@ -14,25 +14,28 @@
 # ==============================================================================
 """Additional optimizers that conform to Keras API."""
 
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
+from tensorflow_addons.optimizers.average_wrapper import AveragedOptimizerWrapper
 from tensorflow_addons.optimizers.conditional_gradient import ConditionalGradient
+from tensorflow_addons.optimizers.cyclical_learning_rate import CyclicalLearningRate
 from tensorflow_addons.optimizers.cyclical_learning_rate import (
-    CyclicalLearningRate)
-from tensorflow_addons.optimizers.cyclical_learning_rate import (
-    TriangularCyclicalLearningRate)
+    TriangularCyclicalLearningRate,
+)
 from tensorflow_addons.optimizers.cyclical_learning_rate import (
-    Triangular2CyclicalLearningRate)
+    Triangular2CyclicalLearningRate,
+)
 from tensorflow_addons.optimizers.cyclical_learning_rate import (
-    ExponentialCyclicalLearningRate)
+    ExponentialCyclicalLearningRate,
+)
 from tensorflow_addons.optimizers.lamb import LAMB
+from tensorflow_addons.optimizers.momentum_lars import MomentumLARS
```

Comment:
Please place these in alphabetical order.

Reply:
done